tests/compilation_tests/grid_test.py

```diff
@@ -135,6 +135,73 @@ def test_is_on_border(self):
 
         self.assertEqual(len(g.border_nodes), count)
 
+    def test_ActivateRectangle(self):
+        lat_min, lat_max, long_min, long_max = 42.444250, 42.444599, -76.483682, -76.483276
+        g = Grid(lat_min, lat_max, long_min, long_max)
+        
+        my_border = []
+        my_border += list(g.nodes[0][1:-1])
+        my_border += list(g.nodes[38][1:-1])
+        my_border += list(g.nodes[: , 0])
+        my_border += list(g.nodes[: , 33])
+        
+        rows = g.nodes.shape[0]
+        cols = g.nodes.shape[1]
+
+        g.activate_rectangle(0, 0, rows, cols)
+        g.find_border_nodes()
+
+        #Uncomment to plot borders and compare
+
+        # for node in my_border:
+        #     plt.plot(node.x, node.y, marker = '.')
+
+        # for node in g.border_nodes:
+        #     plt.plot(node[1], node[2], marker = '.')
+        
+        #plt.show()
+        
+        list_bordernodes = set([(node[0].x,node[0].y) for node in g.border_nodes])
+        my_border = set([(node.x,node.y) for node in my_border])
+
+        self.assertEqual(my_border, list_bordernodes)
+
+
+    def test_ActivateTriangle(self):
+
+        lat_min, lat_max, long_min, long_max = 42.444250, 42.444599, -76.483682, -76.483276
+        g = Grid(lat_min, lat_max, long_min, long_max)
+        
+        my_border = []
+        my_border += list(np.diagonal(g.nodes[:17]))
+        my_border += list(g.nodes[0][:-1])
+        for x in range(16, 33):
```

Comment:
because there are hard-coded numbers, let's add a quick comment on how you arrived at them
maybe something like "Based on the given latitude and longitudes, node 16 on the x axis is the midpoint" or something